.pre-commit-config.yaml

```
@@ -10,7 +10,7 @@ repos:
   - id: isort
 
 - repo: https://github.com/PyCQA/flake8
-  rev: 5.0.4
+  rev: 6.0.0
```

Revert flake8 version because of flake8-isort 4.2.0 depends on flake8<6 and >=3.2.1 .
```suggestion
  rev: 5.0.4
```